test/test_s3.py

```diff
@@ -260,6 +260,8 @@ def setUp(self):
         self.done_error_headers = None
         self.done_error_body = None
         self.done_error_operation_name = None
+        self.did_validate_checksum = None
+        self.checksum_validation_algorithm = None
```

Comment:
naming pattern is `self.done_XYZ` for stuff that comes from the on_done callback